doc/sphinx/notebooks/advanced/natural_gradients.pct.py

```diff
@@ -52,7 +52,7 @@
 autotune = tf.data.experimental.AUTOTUNE
 
 # %% [markdown]
-# ### VGP is a GPR
+# ## VGP is a GPR
```

Comment:
Sphinx complained something something an inconsistent header in this file, so I redid all of them. :shrug: